src/Sarif.Multitool/Rules/RuleResources.resx

```diff
@@ -204,4 +210,7 @@
   <data name="SARIF1007_RegionPropertiesMustBeConsistent_FullDescription_Text" xml:space="preserve">
     <value>Placeholder_SARIF1007_RegionPropertiesMustBeConsistent_FullDescription_Text</value>
   </data>
+  <data name="SARIF1008_PhysicalLocationPropertiesMustBeConsistent_Error_ContextRegionMustBeProperSupersetOfRegion_Text" xml:space="preserve">
+    <value>{0}: This 'physicalLocation' object contains both a 'region' and a 'contextRegion' property, but 'contextRegion' is not a proper superset of 'region'. This is invalid because the purpose of 'contextRegion' is to provide a viewing context around the 'region' which is the location of the result. If the tool simply reversed 'region', then fix it puts the correct values in the correct properties. If 'region' and 'contextRegion' are identical, the 'contextRegion' is unnecessary, and (by the spec) the tool must not populate it.</value>
```

Comment:
>, [](start = 342, length = 1)

My typo. Should be "... simply reversed 'region' and 'contextRegion',".

Reply:
oh my! :) i should have noticed it! thank u, will fix 

---
In reply to: [443860795](https://github.com/microsoft/sarif-sdk/pull/1925#discussion_r443860795) [](ancestors = 443860795)

Reply:
PS: wats "fix it" ? is it some tool? re: then fix it puts the correct values in the correct properties.

---
In reply to: [443861877](https://github.com/microsoft/sarif-sdk/pull/1925#discussion_r443861877) [](ancestors = 443861877,443860795)

Reply:
Yes, "fix it" refers to the tool that produced that bad SARIF. And "then fix it so puts" is another typo. I meant "then fix it so it puts", but probably "then modify the tool so it puts" is better.

---
In reply to: [443862647](https://github.com/microsoft/sarif-sdk/pull/1925#discussion_r443862647) [](ancestors = 443862647,443861877,443860795)

Reply:
Yes, let's make that change.

---
In reply to: [443868912](https://github.com/microsoft/sarif-sdk/pull/1925#discussion_r443868912) [](ancestors = 443868912,443862647,443861877,443860795)